src/foundry/client/documents/user.d.mts

```diff
@@ -653,10 +703,20 @@ declare class User extends BaseUser.Internal.ClientDocument {
 
   /**
    * Update the set of Token targets for the user given an array of provided Token ids.
+   * This function handles changes made elsewhere and does not broadcast to other connected clients.
    * @param targetIds - An array of Token ids which represents the new target set (default: `[]`)
    */
   // targetIds: not null (parameter default only)
-  updateTokenTargets(targetIds?: string[]): void;
+  protected _onUpdateTokenTargets(targetIds?: string[]): void;
+
+  /**
+   * Query this user
+   * @param queryName    - The query name (must be registered in `CONFIG.queries`)
+   * @param queryData    - The query data (must be JSON-serializable)
+   * @param queryOptions - The query options
+   * @returns The query result
+   */
+  query<QueryName extends User.QueryName>(queryName: QueryName, queryData: User.QueryData<QueryName>, {timeout}?: User.QueryOptions): Promise<User.QueryReturn<QueryName>>;
```

Comment:
I bet this should be `Promise<Awaited<User.QueryReturn<QueryName>>>`. Makes me think this should be inlined to `QueryReturn` though.